### PR TITLE
feat: Implement internationalization

### PR DESCRIPTION
**Pull request by danielptv**
[ODDSK-88](https://h-ka-team-rdqzrlfpomci.atlassian.net/browse/ODDSK-88).

Working examples:
- English: https://deploy-preview-30--open-data-dashboard.netlify.app/en/resource/1
- German: https://deploy-preview-30--open-data-dashboard.netlify.app/de/resource/1

The LocaleSwitcher is temporarily included in the ChartTableFilterHead component for testing purposes.

Example not-found page: https://deploy-preview-30--open-data-dashboard.netlify.app/en/resource/10.

### CODE REVIEW

File: src/messages/de.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "Table": {
+    "rowsPerPageText": "Zeilen pro Seite:",
+    "rangeSeparatorText": "von",
+    "selectAllRowsItemText": "Alle"
+  },
+  "LocaleSwitcher": {
+    "locale": "{locale, select, de {🇩🇪 Deutsch} en {🇬🇧 English} other {Unbekannt}}"
+  },
+  "NotFound": {
+    "title": "Hoppla! Du bist verloren.",
```
Comment (Zerdados) [inline src/messages/de.json:24]: Maybe `Hoppla! Du hast dich verirrt.` instead?

File: src/messages/en.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "rangeSeparatorText": "of",
+    "selectAllRowsItemText": "All"
+  },
+  "LocaleSwitcher": {
+    "locale": "{locale, select, de {🇩🇪 Deutsch} en {🇬🇧 English} other {Unknown}}"
+  },
+  "NotFound": {
+    "title": "Oops! You're lost.",
+    "subtitle": "The page you are looking for was not found.",
+    "returnBtn": "Back to Home"
```
Comment (Zerdados) [inline src/messages/en.json:26]: Change this to `Return Home`, or `Return to Homepage` or something like that. `Back to Home` sounds weird.

**Review by Zerdados — COMMENTED** [review]
Generally nice, I just noticed a few things while testing this (I don't believe that the reason for them is in your changes, they just become much more noticeable):
1. When I switch locale or reload the page, I always return to the first tab (In your example the first bar chart)
2. When I open the filter without putting something in, and then reload the page or switch locale, the filter is closed again.

I believe this is because we do not remember this sort of layout information. Perhaps we should add it to our state somehow?

**Comment by danielptv** [comment]
> Generally nice, I just noticed a few things while testing this (I don't believe that the reason for them is in your changes, they just become much more noticeable):
> 
> 1. When I switch locale or reload the page, I always return to the first tab (In your example the first bar chart)
> 2. When I open the filter without putting something in, and then reload the page or switch locale, the filter is closed again.
> 
> I believe this is because we do not remember this sort of layout information. Perhaps we should add it to our state somehow?

Regarding 1.: This functionality is indeed not yet implemented and is tracked in [ODDSK-147](https://h-ka-team-rdqzrlfpomci.atlassian.net/browse/ODDSK-147).

Regarding 2.: Currently it is expected behaviour. The filter is only opens if a filter is set. This will get reviewed and redesigned in [ODDSK-148](https://h-ka-team-rdqzrlfpomci.atlassian.net/browse/ODDSK-148).

**Review by Zerdados — APPROVED** [review]

**Review by ChiaraScheurer — APPROVED** [review]

**Pull request merged**